src/include/daos_security.h

```diff
@@ -246,6 +247,17 @@ daos_acl_remove_ace(struct daos_acl **acl,
 void
 daos_acl_dump(struct daos_acl *acl);
 
+/**
+ * Parse and sanity check the entire Access Control List for valid values and
+ * internal consistency.
+ *
+ * \param	acl	Access Control List to sanity check
+ *
+ * \return	True if the ACL is valid, false otherwise
+ */
+bool
+daos_acl_validate(struct daos_acl *acl);
```

Comment:
since this returns true/false, then daos_acl_is_valid() is a better name.

Reply:
Good idea, I will change the name.